### PR TITLE
Release v0.7.0

### PR DESCRIPTION
**Pull request by alindima**
Signed-off-by: alindima <alindima@amazon.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Changelog
 ## [Unreleased]
 
+## [v0.7.0]
+
+### Changed
```
Comment (lauralt) [inline CHANGELOG.md:6]: nit: newline

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+- [[#178]](https://github.com/rust-vmm/vm-memory/pull/178): `MmapRegion::build_raw` no longer requires that
+  the length is a multiple of the page size.
```
Comment (lauralt) [inline CHANGELOG.md:8]: nit: I would make this a bit clearer, e.g. `length of the region is ...`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
 # Changelog
 ## [Unreleased]
 
+## [v0.7.0]
```
[the remark this answers is not in the document]
Comment (alindima) [reply CHANGELOG.md:4]: done

File: CHANGELOG.md (hunk 1 above)
Comment (alindima) [reply CHANGELOG.md:6]: done

**Review by lauralt — APPROVED** [review]

**Review by alexandruag — APPROVED** [review]

**Pull request merged**